script.js

```diff
@@ -1,4 +1,74 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  var spsOutput = ['scissors', 'paper', 'stone']
+  var setSpsOutput =  Math.floor(Math.random()*spsOutput.length)
+  var comChoice = spsOutput[setSpsOutput]
+  console.log (comChoice)
+
+  if (input == comChoice) {
+    var myOutputValue = "It's a draw."
+  } 
+
+  else if (input == 'reversed scissors' && comChoice == 'scissors' || input == 'reversed paper' && comChoice == 'paper'|| input == 'reversed stone' && comChoice == 'stone') {
+    var myOutputValue = "It's a draw."
+  }
+
+  else if (input == 'scissors') {
+    if (comChoice == 'paper') {
+      var myOutputValue = "You won!"
+    }
+    else if (comChoice == 'stone') {
+      var myOutputValue = "You lost!"
+    }
+  }   
+
+  else if (input == 'reversed scissors') {
+    if (comChoice == 'paper') {
+      var myOutputValue = "You lost!"
+    }
+    else if (comChoice == 'stone') {
+      var myOutputValue = "You won!"
+    }
+  }   
+
+  else if (input == 'paper') {
+    if (comChoice == 'stone') {
+      var myOutputValue = "You won!"
+    }
+    else if (comChoice == 'scissors') {
+      var myOutputValue = "You lost!"
+    }
+  }   
+
+  else if (input == 'reversed paper') {
```

Comment:
I am pretty sure you nailed down all the conditions appropriately well, do insert comments for some of these chunks, will help when you review them in future as well!